pymc3/distributions/distribution.py

```diff
@@ -2,17 +2,21 @@
 import theano.tensor as tt
 from theano import function
 
+from . import transforms
```

Comment:
that was introduced https://github.com/pymc-devs/pymc3/pull/1625, and tests passed (https://coveralls.io/builds/9484082)